client/src/main/java/hudson/plugins/swarm/Client.java

```diff
@@ -190,4 +187,70 @@ public void run(SwarmClient swarmClient, String... args) throws InterruptedExcep
             swarmClient.sleepSeconds(waitTime);
         }
     }
+
+    private static void logArguments(CmdLineParser parser) {
+        Options defaultOptions = new Options();
+        CmdLineParser defaultParser = new CmdLineParser(defaultOptions);
+
+        StringBuilder sb =
+                new StringBuilder("Swarm Client is running with the following configuration: ");
```

Comment:
With your latest change, only logging the provided commandline parameters, this text has to be updated, e.g. back to "Client.main invoked with: ".

Reply:
Done.